src/components/Modals/Send/views/Address.tsx

```diff
@@ -20,20 +20,25 @@ import { useChainAdapters } from 'context/ChainAdaptersProvider/ChainAdaptersPro
 import { useModal } from 'context/ModalProvider/ModalProvider'
 
 import { AddressInput } from '../AddressInput/AddressInput'
-import { SendFormFields } from '../Form'
+import { SendFormFields, SendInput } from '../Form'
 import { SendRoutes } from '../Send'
 
 export const Address = () => {
   const history = useHistory()
   const translate = useTranslate()
   const {
     formState: { errors }
-  } = useFormContext()
-  const [address, asset] = useWatch({ name: [SendFormFields.Address, SendFormFields.Asset] })
+  } = useFormContext<SendInput>()
+  const address = useWatch<SendInput, SendFormFields.Address>({ name: SendFormFields.Address })
+  const asset = useWatch<SendInput, SendFormFields.Asset>({ name: SendFormFields.Asset })
```

Comment:
using `name` as an array has problems with typing, the values are typed as `never`. After some research it looks like not using the array syntax for `name` is a reasonable workaround. The other option is just to watch the entire form.